modules/ROOT/pages/indexes/search-performance-indexes/overview.adoc

```diff
@@ -0,0 +1,24 @@
+:description: Overview of the search-performance indexes available in Neo4j.
+= Search-performance indexes
+
+Search-performance indexes enable quicker retrieval of exact matches between an index and the primary data storage.
+There are four different search-performance indexes available in Neo4j:
+
+* *Range indexes*: Neo4j’s default index.
+Supports most types of predicates, including node label and relationship types.
```

Comment:
```suggestion
Supports most types of predicates, including node labels and relationship types.
```

Reply:
Actually - I think it could just say "Supports most types of predicates"

Reply:
Sure, I'm always in favor of shortening :D